sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -2075,6 +2078,8 @@ static ReadChangeStream create() {
 
     abstract @Nullable Boolean getCreateOrUpdateMetadataTable();
 
+    abstract @Nullable Duration getBacklogReplicationAdjustment();
```

Comment:
I'd prefer to leave this as is for consistency with other optional parameters. We already do a lot of defaulting in ReadChangeStreamIO.expand instead of pushing down optional params into the DoFns and setting defaults there. 

I don't have a strong preference for either approach but I think we should keep this consistent with the other parameters.